lib/common_test/test/ct_surefire_SUITE_data/surefire_SUITE.erl

```diff
@@ -76,12 +77,17 @@ break(_Config) ->
     test_server:break(""),
     ok.
 
-tc_ok(_Config) ->
-    ok.
+tc_ok(Config) when is_list(Config) ->
+    ok;
+tc_ok(_) ->
```

Comment:
Whats the purpose of this clause?

Reply:
To check that the line number reported is the first clause of a function and that multiple clauses actually work.

Reply:
Does it also work if you reverse the clause order? So the 2nd matches?

Reply:
No, it takes the first clause no matter what.